packages/idos-sdk-js/src/lib/enclave-providers/iframe-enclave.ts

```diff
@@ -111,6 +111,12 @@ export class IframeEnclave implements EnclaveProvider {
   }
 
   async #loadEnclave() {
+    const hasIframe = document.getElementById(this.iframe.id);
+    if (hasIframe) {
+      console.warn("An Iframe already exists in the container");
+      return new Promise((resolve) => this.iframe.addEventListener("load", resolve));
```

Comment:
This promise seems like it will never resolve because the load event won't be triggered after it's already been loaded. Am I missing something?

Reply:
yea ur right the event already been set from the first call no need to do it again. 
I gues this does the job
``
return Promise.resolve(null);
``